src/protocols/memory/ddr4.stanza

```diff
@@ -330,108 +460,159 @@ public defmethod constrain (cst:DDR4-Constraint, src:JITXObject, dst:JITXObject)
     val ilane  = width-to-lane(width(cst))
     val irank  = rank-to-int(rank(cst))
 
-    val cst-ck  = DiffPair-Constraint(skew = skew-ck(cst), loss = loss(cst), route-struct = diff-ck-rs(cst))
-    val cst-dqs = DiffPair-Constraint(skew = skew-dqs(cst), loss = loss(cst), route-struct = diff-dqs-rs(cst))
+    constrain(data-cst(cst), src.data, dst.data)
+    constrain(acc-cst(cst), src.acc, dst.acc)
+    constrain(data-acc-cst(cst), src, dst)
 
-    val guide-ck  = src.ctl.CK.P => dst.ctl.CK.P
-    constrain(cst-ck, src.ctl.CK, dst.ctl.CK)
 
-    ; ex. for x8,  i goes from 0 to 1, so all of the byte lanes are individually constrained
-    ; ex. for x16, i goes from 0 to 2, so all of the byte lanes are individually constrained
-    ; ex. for x72, i goes from 0 to 9, so all of the byte lanes are individually constrained
+public defn connect-DDR4 (src:JITXObject, dst:JITXObject
```

Comment:
I'm assuming that this is a point to point connection right - this can't possibly setup a Flyby. The topology requires additional handling.

Reply:
Add documentation